package.json

```diff
@@ -29,6 +29,7 @@
     "cypress-multi-reporters": "^1.6.1",
     "cypress-parallel": "0.12.0",
     "cypress-wait-until": "^1.7.2",
+    "cy-verify-downloads": "^0.1.11",
```

Comment:
Check how big the support community is on this.

Reply:
Had a look at this before deciding to use it - 2.2m downloads - recent commits - is present in cypress's own plugins page in the docs.